app/components/channel_drawer/channel_list.js

```diff
@@ -140,6 +142,10 @@ export default class ChannelList extends React.Component {
         this.props.closeChannelDrawer();
     };
 
+    handleClose = (channel) => {
```

Comment:
I don't know if we need that since it's not destructive. If you accidentally remove someone, you can just re-add them from the More list. I'll bring it up with UX, but we can merge this in the meantime